kafka_consumer/datadog_checks/kafka_consumer/client.py

```diff
@@ -8,6 +8,8 @@
 
 from datadog_checks.kafka_consumer.constants import KAFKA_INTERNAL_TOPICS, OFFSET_INVALID
 
+LIBRDKAFKA_LOG_CRIT = 2
```

Comment:
Could you add a link to some kafka docs that explain these log levels?

Reply:
I did not find any docs, had to read the code so I linked the file

Reply:
lovely 😂